SDL/Kernels.cuh

```diff
@@ -40,30 +41,19 @@ const unsigned int N_MAX_SEGMENTS_PER_MODULE = 600; //Also used in (pixel)trackl
 const unsigned int N_MAX_PIXEL_MD_PER_MODULES = 100000;
 const unsigned int N_MAX_PIXEL_SEGMENTS_PER_MODULE = 50000;
 
-//const unsigned int N_MAX_TRIPLETS_PER_MODULE = 5000;
```

Comment:
Can you keep these in the comments. It might still be useful if we need to increase these values.